src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -56,6 +57,12 @@
 # to store he last timestamp that the tasks were triggered.
 HIGH_VOLUME_LAST_TRIGGER_TS_KEY = "sentry.monitors.last_tasks_ts"
 
+# This delay in seconds defines how long we wait after the minute before
+# running the monitor tasks. These tasks are responsible for creating missed
+# check-ins and marking timedout check-ins. Thus, having some delay after the
+# start of the minute is critical.
+MONITOR_TASK_DELAY = 15
```

Comment:
quick question about this, i know before we changed the logic of the check_missing task to only mark missing tasks that were already a minute past due. E.g. at `11:01pm` it would mark the `11:00pm` expected scheduled cron job as missing. Do we just want this extra 15 seconds as additional safety or has some other logic changed along with this?

Reply:
Did we actually do that?